### DISCUSSION TITLE
Match a word in a sentence

### DISCUSSION THREAD

**Discussion by GerhardusOtto** [pallets/jinja#1887] [Q&A]
Good day,
I was hoping someone could assist me.
I have the following situation.

I am receiving the following payload.
"sender" email address
"message" context 
"subject" context

In the message field there is either unregistered or registered mentioned at some point.

I am currently using the following expression:
{{payload.message == 'registered'}} which reports true if registered is the only word in the message field, however if it is part of a sentence it reports as false.

Is there any way to have the statement report true if the word is present at all in the message body

Thank you kindly to anyone who can help

**Comment by ThiefMaster** [comment]
Maybe check for `'registered' in payload.message`? It would not check specifically for a word (so e.g. "unregistered" would also match), but probably fine for whatever you're trying to do.
